LLM/embedding/deploy-embedding-model-tei.md

```diff
@@ -0,0 +1,268 @@
+# Overview
```

Comment:
SHouldn't we use more explicit header here? Something like "Deploying embedding models with TEI on OCI MD"?